crates/swc_ecma_parser/tests/typescript/class/override/input.ts

```diff
@@ -0,0 +1,5 @@
+class C extends B {
+  override prop = 5;
+  override method() {
```

Comment:
I got a bit confused because the override keyword didn't show up expect when doing `extends`. This seems ok, but probably the AST should just represent the code as-is.